R/03-geojob-obj.R

```diff
@@ -77,7 +77,7 @@ setMethod("geojob", signature("XMLDocument"), function(xml, ...) {
 #'@aliases geojob,geojob-method
 setMethod("geojob", signature("character"), function(xml, ...) {
   #parse based on xml class
-  if(startsWith(xml, "http")){
+  if(length(xml == 1) && startsWith(x = xml, prefix= "http")){
```

Comment:
We should probably make an issue of what we actually want to handle here and make it more robust, but I think it is fine for now.